tests_e2e.py

```diff
@@ -0,0 +1,366 @@
+# *** NATIVE DEPENDENCIES ***
+import threading
+import random
+import json
+import urllib.parse
+from http.cookies import SimpleCookie
+
+# *** LIBRARIES ***
+import pytest
+import unittest
+import requests
+
+# *** HEDY RESOURCES ***
+import utils
+from config import config as CONFIG
+
+# *** GLOBAL VARIABLES ***
+
+HOST = 'http://localhost:' + str (CONFIG ['port']) + '/'
+# This dict has global scope and holds all created users and their still current sessions (as cookies), for convenient reuse wherever needed
+USERS = {}
```

Comment:
Do we clean up these users at the end of the test? For the local database I don't care, but otherwise this would accumulate endless users on the `alpha` database and that concerns me a little.